develop/dev-guide-proxysql-integration.md

```diff
@@ -104,579 +655,785 @@ The production environment requires more steps than the test environment. To dep
 
-After adding a TiDB server user to ProxySQL, ProxySQL allows this user to log in **_ProxySQL MySQL Interface_** and create a connection with TiDB. Make sure that the user has appropriate permissions in TiDB. To add a TiDB server user, perform the following at **_ProxySQL Admin interface_**:
+1. Delete any existing SQLite database (where configurations are stored internally):
 
-```sql
-INSERT INTO mysql_users(username, password, active, default_hostgroup, transaction_persistent) VALUES ('root', '', 1, 0, 1);
-LOAD mysql users TO runtime;
-SAVE mysql users TO DISK;
-```
+    ```bash
+    rm /var/lib/proxysql/proxysql.db
+    ```
 
-Field description:
+    > **Warning:**
+    >
+    > If you delete the SQLite database file, any configuration changes made using ProxySQL Admin interface will be lost.
 
-- `username`: The user name.
-- `password`: The password.
-- `active`: Controls whether the user is active. `1` is active, and `0` is inactive. Only when the `active` is `1`, the user can log in.
-- `default_hostgroup`: The default hostgroup used by the user, where SQL distributed to unless the query rule routes the traffic to a specific hostgroup.
-- `transaction_persistent`: `1` indicates persistent transaction. That is, when the user starts a transaction in a connection, all statements are routed to the same hostgroup until the transaction is committed or rolled back.
+2. Locate the configuration file in `/etc/proxysql.cnf` and modify it according to your need.
 
-### Configure ProxySQL by a configuration file
+    For example:
 
-In addition to configuring using **_ProxySQL Admin interface_**, you can also configure ProxySQL using a configuration file. In the [Configuring ProxySQL through the config file](https://github.com/sysown/proxysql#configuring-proxysql-through-the-config-file) document, the configuration file should only be considered as a secondary way of initializing ProxySQL, not the primary way. The configuration file is only used when the SQLite is not created and will not be used after the SQLite is created. When using the configuration file to configure ProxySQL, you should delete SQLite first using the following command. But this **loses** configuration changes in **_ProxySQL Admin interface_**.
+    ```
+    mysql_servers:
+    (
+        {
+            address="<tidb cloud dedicated cluster host>"
+            port=<tidb cloud dedicated cluster port>
+            hostgroup=0
+            max_connections=2000
+        }
+    )
+
+    mysql_users:
+    (
+        {
+            username = "<tidb cloud dedicated cluster username>"
+            password = "<tidb cloud dedicated cluster password>"
+            default_hostgroup = 0
+            max_connections = 1000
+            default_schema = "test"
+            active = 1
+            transaction_persistent = 1
+        }
+    )
+    ```
 
```

Comment:
@Icemap add a note section here 

Replace `tidb cloud dedicated cluster host` with the actual TiDB Cloud `endpoint`
`tidb cloud dedicated cluster username` with TiDB Cloud `username`
`tidb cloud dedicated cluster password` with TiDB Cloud `password`